site/src/docs/components/date-input/customisation.mdx

```diff
@@ -0,0 +1,77 @@
+---
+slug: '/components/date-input/customisation'
+title: 'DateInput - Customisation'
+---
+
+import { useState } from 'react';
+import { DateInput } from 'hds-react';
+import TabsLayout from './tabs.mdx';
+
+export default ({ children, pageContext }) => <TabsLayout pageContext={pageContext}>{children}</TabsLayout>;
+
+## Customisation
+
+### Customisation properties
+The component property `setDateClassNames` can be used to set class names for specific dates.
+
+### Customisation example
+In this example some days have different background colours to indicate that they have limited timeslots available. When days have colours to indicate something, it's important to include `legend` and `setDateAriaDescribedBy` props to inform the users of their meaning. Use the `setDateAriaDescribedBy` callback function to return corresponding LegendItem's element id for specific dates.  
+
+<Playground scope={{ DateInput, useState }}>
+
+
+  ```jsx
+import { DateInput } from 'hds-react';
+
+() => {
+    const dayHasSomeSpace = (day) => day === 12 || day === 15 || day === 24 || day === 25;
+    const dayHasLittleSpace = (day) => day === 3 || day === 4 || day === 17 || day === 19;
+
+    const setDateClassNames = (date) => {
+        const day = date.getDate();
```

Comment:
![image](https://github.com/City-of-Helsinki/helsinki-design-system/assets/2639230/946d5fd7-a145-42e2-a525-91584786677e)

I'm not sure is the demo old version or broken

Reply:
Forgot to update docs, now fixed.

Reply:
Forgot to update demo though 🤦 Will do later.

Reply:
Demo updated.